py/persistentcode.c

```diff
@@ -408,12 +408,15 @@ STATIC mp_raw_code_t *load_raw_code(mp_reader_t *reader, qstr_window_t *qw) {
     }
 
     mp_uint_t *const_table = NULL;
+    size_t n_obj = 0;
+    size_t n_raw_code = 0;
```

Comment:
Since `n_obj` and `n_raw_code` are set before `const_table`, it makes sense to put their declarations here before it, to keep the logical order of these variables.